docs/src/main/paradox/configuration.md

```diff
@@ -1,6 +1,9 @@
 # Configuration
 
-Just like any other Akka module Akka HTTP is configured via [Typesafe Config](https://github.com/lightbend/config).
+Before configuring Akka HTTP it must be added to your project as described in
+@ref[the introduction](introduction.md#using-akka-http)
+
+Just like any other Akka module Akka HTTP is configured via [Lightbend Config](https://github.com/lightbend/config).
```

Comment:
Isn't the library still called "typesafe config" though?

Reply:
Oops! Not sure what I was thinking.